package.json

```diff
@@ -73,10 +73,9 @@
     "enzyme-adapter-react-16": "^1.15.2",
     "enzyme-to-json": "^3.4.4",
     "eslint": "^6.8.0",
-    "eslint-config-airbnb": "^18.0.1",
     "eslint-config-prettier": "^6.10.0",
     "eslint-import-resolver-alias": "^1.1.2",
-    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
Is [comment](https://github.com/k-bialucha/business-assistant-mobile-app/blob/58d1279f292674590b0c860d7609aa655f41c5db/.eslintrc.js#r555335687) above connected with updating this package? :thinking:

Reply:
I can't see a reference to the comment :disappointed: But I guess it was just regular package update to enable new features (or something was fixed after this update :smile:).

Reply:
I thought it is something about added `'import'`, in `.eslintrc.js` file